agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -458,6 +458,12 @@ bool main_thread::finalize_slaves_connect_state(bool fConnected,
                 continue;
             }
```

Comment:
Regarding This commit which fix double sending of backhaul connected notification.
I already had a fix for it on Intel Bitbucket. See commit 22956a4d3a6, 23 April 2019, and lock for:
"soc->backhaul_operational_on_registration".

Reply:
ok, I didn't know. I will take your change instead - thanks.